SQL/beestation_schema.sql

```diff
@@ -81,37 +81,43 @@ CREATE TABLE IF NOT EXISTS `SS13_ban` (
-  `gender` varchar(50) NOT NULL,
-  `age` tinyint(3) unsigned NOT NULL,
-  `hair_color` varchar(50) NOT NULL,
-  `facial_hair_color` varchar(50) NOT NULL,
-  `eye_color` varchar(50) NOT NULL,
-  `skin_tone` varchar(50) NOT NULL,
-  `hair_style_name` varchar(50) NOT NULL,
-  `facial_style_name` varchar(50) NOT NULL,
-  `feature_ethcolor` varchar(50) NOT NULL,
-  `underwear` varchar(50) NOT NULL,
-  `undershirt` varchar(50) NOT NULL,
-  `socks` varchar(50) NOT NULL,
-  `backbag` varchar(50) NOT NULL,
-  `uplink_loc` varchar(50) NOT NULL,
-  `species` varchar(50) NOT NULL,
-  `features` json NOT NULL,
-  `joblessrole` smallint(6) NOT NULL,
-  `job_civlian_high` smallint(6) NOT NULL,
-  `job_civilian_med` smallint(6) NOT NULL,
-  `job_civilian_low` smallint(6) NOT NULL,
-  `job_medsci_high` smallint(6) NOT NULL,
-  `job_medsci_med` smallint(6) NOT NULL,
-  `job_medsci_low` smallint(6) NOT NULL,
-  `job_engsec_high` smallint(6) NOT NULL,
-  `job_engsec_med` smallint(6) NOT NULL,
-  `job_engsec_low` smallint(6) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+CREATE TABLE `SS13_characters` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`slot` INT(11) NOT NULL,
+	`ckey` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`species` VARCHAR(32) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`real_name` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`name_is_always_random` BIT(1) NOT NULL,
+	`body_is_always_random` BIT(1) NOT NULL,
+	`gender` VARCHAR(16) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`age` TINYINT(3) UNSIGNED NOT NULL,
+	`hair_color` VARCHAR(8) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`gradient_color` VARCHAR(8) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`facial_hair_color` VARCHAR(8) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`eye_color` VARCHAR(8) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`skin_tone` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`hair_style_name` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`gradient_style` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`facial_style_name` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`underwear` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`underwear_color` VARCHAR(32) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`undershirt` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`socks` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`backbag` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`jumpsuit_style` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`uplink_loc` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`features` MEDIUMTEXT NOT NULL COLLATE 'utf8mb4_general_ci',
+	`custom_names` MEDIUMTEXT NOT NULL COLLATE 'utf8mb4_general_ci',
+	`helmet_style` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`preferred_ai_core_display` VARCHAR(64) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`preferred_security_department` VARCHAR(32) NOT NULL COLLATE 'utf8mb4_general_ci',
+	`joblessrole` TINYINT(4) NOT NULL,
```

Comment:
same here, any reason why is it not unsigned?

Reply:
No